### PR TITLE
changed to-float to to-decimal

### PR DESCRIPTION
**Pull request by routrohan**
Changed sub-command to-float to to-decimal to maintain naming convention in the project. Also made necessary changes to dependencies and file names accordingly.

### CODE REVIEW

File: crates/nu-cli/src/commands/str_/to_decimal.rs [hunk 1]
```diff
@@ -21,11 +21,11 @@ pub struct SubCommand;
...
     fn name(&self) -> &str {
-        "str to-float"
+        "str to-decimal"
     }
 
     fn signature(&self) -> Signature {
-        Signature::build("str to-float").rest(
+        Signature::build("str to-decimal").rest(
             SyntaxShape::ColumnPath,
             "optionally convert text into float by column paths",
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/str_/to_decimal.rs:30]: may also want to change this `float` to `decimal`

File: crates/nu-cli/src/commands/str_/to_decimal.rs [hunk 2]
```diff
@@ -46,7 +46,7 @@ impl WholeStreamCommand for SubCommand {
     fn examples(&self) -> Vec<Example> {
         vec![Example {
             description: "Convert to float",
```
Comment (jonathandturner) [inline crates/nu-cli/src/commands/str_/to_decimal.rs:48]: and this `float`

Comment (jonathandturner) [reply crates/nu-cli/src/commands/str_/to_decimal.rs:48]: Looks like there might be 3 total in the file

Comment (routrohan) [reply crates/nu-cli/src/commands/str_/to_decimal.rs:48]: Alright
on it

**Comment by andrasio** [comment]
Looks good!

**Comment by routrohan** [comment]
@jonathandturner please let me know if it's fine now

**Comment by jonathandturner** [comment]
Thanks!

**Pull request merged**